app/AppInit.jsx

```diff
@@ -55,7 +55,24 @@ class AppInit extends React.Component {
         };
     }
 
+    componentDidCatch(error, errorInfo) {
+        LogsActions.setLog({error, errorInfo});
+    }
+
     componentWillMount() {
+        console.log = function() {
```

Comment:
Duplicated implementation should be declared only once. Also I think we could save info about loglevel used in logs array, what do you think?